### ISSUE TITLE
kernel_protoshares launch failed: invalid device function

### ISSUE THREAD

**Issue by jfhutchi** [dave-andersen/cudapts#1]
On Geforce 550 Ti

**Comment by dave-andersen** [comment]
What CUDA version are you compiling with and what platform (linux?) are you running on?

**Comment by jfhutchi** [comment]
Windows 8.1 Pro 64bit using Cygwin 64bit. CUDA 5.5

**Comment by spommerening** [comment]
If I can support you with any testing, please let me know. Unfortunately I have near zero knowledge in Windows programming (Linux DevOps), but my gaming pc is running Windows 8.1 and has an Nvidia Geforce GTX 770 graphic card. And of course I am also very interested in getting this thing stable on Windows. ;-)

**Comment by jfhutchi** [comment]
yeah basically I'm in the same boat this is my gaming pc..(little old ;) )

On Fri, Jan 10, 2014 at 6:10 PM, Stefan Pommerening <
notifications@github.com> wrote:

> If I can support you with any testing, please let me know. Unfortunately I
> have near zero knowledge in Windows programming (Linux DevOps), but my
> gaming pc is running Windows 8.1 and has an Nvidia Geforce GTX 770 graphic
> card. And of course I am also very interested in getting this thing stable
> on Windows. ;-)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/dave-andersen/cudapts/issues/1#issuecomment-32076321
> .

## 

John F. Hutchinson
